cmd/car/car.go

```diff
@@ -71,18 +71,25 @@ func main1() int {
 				}},
 			},
 			{
-				Name:    "extract",
-				Aliases: []string{"x"},
-				Usage:   "Extract the contents of a car when the car encodes UnixFS data",
-				Action:  ExtractCar,
+				Name:      "extract",
+				Aliases:   []string{"x"},
+				Usage:     "Extract the contents of a car when the car encodes UnixFS data",
+				Action:    ExtractCar,
+				ArgsUsage: "[output directory|-]",
 				Flags: []cli.Flag{
 					&cli.StringFlag{
 						Name:      "file",
 						Aliases:   []string{"f"},
-						Usage:     "The car file to extract from",
+						Usage:     "The car file to extract from, or '-' to read from stdin",
 						Required:  true,
```

Comment:
should we make a not specified (e.g. no -f set on cli) also be interpreted as read from stdin?

Reply:
sure, see latest push for making `-f` optional and reading from stdin in that case

I'm always forgetting `-f` anyway and you get a hostile error, now you get a more gentle note that it's trying to read.